content/tutorials/trainapp-fitbit.textile

```diff
@@ -0,0 +1,550 @@
+---
+title: Building a Clockface app for Fitbit using Ably
+section: tutorials
+index: 95
+---
+
+In this tutorial, we will see how to use the "Ably Realtime client library":/realtime to build a realtime London Tube schedule clockface app for Fitbit. Ably enables realtime data sharing using Pub/Sub messaging architecture via a concept called "channels":https://www.ably.io/channels.
+
+For the purpose of this tutorial, we'll not discuss storage of messages or other server-side mechanisms.
+
+
+"Getting started with Fitbit SDK":https://dev.fitbit.com/getting-started/ is a guide to get started for developing apps and clockface apps for Fitbit.
+
+<%= partial 'tutorials/_step-1-setup-free-account' %>
+
+h2(#create-clockface-app). Step 2 - Create a clock face 
+
+Prerequisites:
+
+* Make sure "Node and NPM are installed":https://nodejs.org/en/download/ on your system.
+* You will not need to install the Fitbit SDK separately as we'll be using NPX (that comes bundled with NPM) for doing this.
+* Fitbit user account. "Sign up here":https://www.fitbit.com/signup.
+* A Fitbit OS device, or the Fitbit OS Simulator for Windows or macOS.
+* The latest Fitbit mobile application for Android, iOS or Windows Phone, paired with your Fitbit device.
+* A computer with access to Fitbit Studio.
+* A Wifi connection to provide the Fitbit device a connection to the internet.
+
+We'll be using the Fitbit command line interface for creating the clockface. You can find the guide "here":https://dev.fitbit.com/build/guides/command-line-interface/ .
+
+We'll start by creating a new clockface. To do this, we'll be using the @create-fitbit-app@ command that allows you to generate the minimal scaffolding required for either an app or clock face.
```

Comment:
clock face at end of sentence with a space between. Should this be `clockface` rather than `clock face`?